hack/tools/e2e-log-sync/README.md

```diff
@@ -0,0 +1,32 @@
+
+# E2e log sync
+
+## Prerequisites
+
+Start the Tilt development environment via `tilt up`.
+
+*Notes*:
+* If you only want to see imported logs, disable promtail.
+* If you want to drop all logs from Loki, just delete the Loki Pod in the `observability` namespace.
+
+## Import logs
+
+Imports logs into Loki:
```

Comment:
I have to test it, but it looks like I can add a Button to the tilt UI to import logs:
https://docs.tilt.dev/buttons.html

Reply:
Done